pytorch_tabnet/utils.py

```diff
@@ -103,24 +105,32 @@ def create_dataloaders(X_train, y_train, X_valid, y_valid, weights,
     else:
         # custom weights
         if len(weights) != len(y_train):
-            raise ValueError('Custom weights should match number of train samples.')
+            raise ValueError("Custom weights should match number of train samples.")
         need_shuffle = False
         samples_weight = np.array(weights)
         sampler = WeightedRandomSampler(samples_weight, len(samples_weight))
 
-    train_dataloader = DataLoader(TorchDataset(X_train, y_train),
-                                  batch_size=batch_size,
-                                  sampler=sampler,
-                                  shuffle=need_shuffle,
-                                  num_workers=num_workers,
-                                  drop_last=drop_last)
-
-    valid_dataloader = DataLoader(TorchDataset(X_valid, y_valid),
-                                  batch_size=batch_size,
-                                  shuffle=False,
-                                  num_workers=num_workers)
-
-    return train_dataloader, valid_dataloader
+    train_dataloader = DataLoader(
+        TorchDataset(X_train, y_train),
```

Comment:
I know it's black auto linting but I really find this harder to read than the previous version. You can obviously ignore this complaint.